.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "pkg/vm/testdata/neo-vm"]
+	path = pkg/vm/testdata/neo-vm
+	url = git@github.com:neo-project/neo-vm.git
```

Comment:
You're better using `https://` URL here because otherwise you make anonymous users not being able to checkout submodule.